mailpoet/assets/js/src/homepage/components/resources.tsx

```diff
@@ -52,7 +56,48 @@ export function Resources(): JSX.Element {
       className="mailpoet-homepage-resources"
       heading={MailPoet.I18n.t('learnMoreAboutEmailMarketing')}
     >
-      {posts.map((post) => post)}
+      <div className="mailpoet-homepage-resources__posts">
+        {posts
+          .filter(
+            (_post, index) =>
+              index + 1 === activePage * 2 || index + 1 === activePage * 2 - 1,
+          )
+          .map((post) => post)}
+      </div>
+      <div className="mailpoet-homepage-resources__pagination">
+        {MailPoet.I18n.t('pageOf')
+          .replace('%1$d', activePage.toString())
+          .replace('%2$d', Math.ceil(posts.length / 2).toString())}
+        {activePage > 1 ? (
+          <a
+            href="#"
+            onClick={(e) => {
+              e.preventDefault();
+              setActivePage(activePage - 1);
+            }}
+            title={MailPoet.I18n.t('previousPostsPage')}
+          >
+            <Icon icon={chevronLeft} />
+          </a>
+        ) : (
+          <Icon icon={chevronLeft} />
+        )}
+
+        {activePage < Math.ceil(posts.length / 2) ? (
+          <a
+            href="#"
+            onClick={(e) => {
+              e.preventDefault();
+              setActivePage(activePage + 1);
```

Comment:
Good point! I found that in order to avoid creating anonymous functions, I needed to add two functions because in the second place I need to set `activePage - 1`, and creating a new function that would accept that as a parameter would mean calling in in an anonymous function. I added those in a new commit.